terraform/deployments/govuk-publishing-platform/security_group_rules.tf

```diff
@@ -261,10 +261,40 @@ resource "aws_security_group_rule" "publisher_to_any_any" {
   security_group_id = module.publisher_web.security_group_id
 }
 
+resource "aws_security_group_rule" "publisher_worker_to_any_any" {
+  description       = "Publisher worker sends requests to anywhere over any protocol"
+  type              = "egress"
+  from_port         = 0
+  to_port           = 0
+  protocol          = -1
+  cidr_blocks       = ["0.0.0.0/0"]
+  security_group_id = module.publisher_worker.security_group_id
+}
+
 #
 # Publishing API
 #
 
+resource "aws_security_group_rule" "publishing_web_to_any_any" {
+  description       = "Publishing web sends requests to anywhere over any protocol"
+  type              = "egress"
+  from_port         = 0
+  to_port           = 0
+  protocol          = -1
+  cidr_blocks       = ["0.0.0.0/0"]
+  security_group_id = module.publishing_api_web.security_group_id
+}
+
+resource "aws_security_group_rule" "publishing_worker_to_any_any" {
```

Comment:
Similarly this is for Publishing API, can we rename it something like `publishing_api_worker_to_any_any`?

Reply:
yes.